content/tutorials/token-authentication.textile

```diff
@@ -494,6 +672,19 @@ blang[ruby].
 
   And then run the demo locally by adding your "Ably API key":https://support.ably.io/support/solutions/articles/3000030502-setting-up-and-managing-api-keys to @server.rb@ and running the demo @bundle exec ruby server.rb@
 
+blang[php].
+  The complete source code for each step of "this tutorial is available on Github":https://github.com/ably/tutorials/commits/token-authentication-php.
+
+  We recommend that you clone the repo locally:
+
+  bc[sh]. git clone https://github.com/ably/tutorials.git
+
+  Checkout the tutorial branch:
+
+  bc[sh]. git checkout token-authentication-php
+
+  And then run the demo locally by adding your "Ably API key":https://support.ably.io/support/solutions/articles/3000030502-setting-up-and-managing-api-keys to @ably.php@ and running the demo @php -S 0.0.0.0:8000@ inside @public@ directory.
```

Comment:
If you are running the demo inside the public directory, is that directory needed?  If it's a convention that you think is important enough to stick with then fine, but I'd like to know your view